src/apps/routes/apps.clj

```diff
@@ -77,6 +77,17 @@
     operation."
     (ok (apps/unshare-apps current-user (:unsharing body))))
 
+  (context "/systems" []
+
+    (GET "/ids" []
+      :query [params SecuredQueryParams]
```

Comment:
Hmm, yeah, the `App endpoints` group might be getting a little bit cluttered. I might create a separate root context instead, though. Reusing the `Service Status Information` group  would work, although I'd either have to modify the endpoint so that it doesn't require user information or add the middleware to the context inside `apps.routes.status`.